collector/pkg/component/analyzer/network/protocol/rocketmq/rocketmq_response.go

```diff
@@ -0,0 +1,53 @@
+package rocketmq
+
+import (
+	"encoding/json"
+	"github.com/Kindling-project/kindling/collector/pkg/component/analyzer/network/protocol"
+	"github.com/Kindling-project/kindling/collector/pkg/model/constlabels"
+)
+
+func fastfailRocketMQResponse() protocol.FastFailFn {
+	return func(message *protocol.PayloadMessage) bool {
+		// 根据报文实现具体的fastFail()函数
+		return len(message.Data) < 8
```

Comment:
Header Length is more than 18, if there is a payload length is 10, parseHeader will indexoutofbound.